meta-lmp-base/recipes-security/optee/optee-client-fio.inc

```diff
@@ -24,7 +25,12 @@ EXTRA_OECMAKE = " \
 EXTRA_OECMAKE:append:toolchain-clang = " -DCFG_WERROR=0"
 
 do_install:append() {
+    install -d ${D}${datadir}/p11-kit/modules
+    install -m 0644 ${WORKDIR}/ckteec.module ${D}${datadir}/p11-kit/modules/ckteec.module
     install -D -p -m0644 ${WORKDIR}/tee-supplicant.service ${D}${systemd_system_unitdir}/tee-supplicant.service
 }
 
 SYSTEMD_SERVICE:${PN} = "tee-supplicant.service"
+FILES:${PN} += " \
+    ${datadir}/p11-kit \
```

Comment:
The directory installed was `${datadir}/p11-kit/modules` so it's better to it here

Reply:
Done.